pos_backend_communication/static/src/js/back.js

```diff
@@ -0,0 +1,47 @@
+odoo.define("pos_backend_communication.back", function (require) {
+    "use strict";
+    var tools = require("pos_backend_communication.tools");
```

Comment:
```suggestion
    const tools = require("pos_backend_communication.tools");
```

Non blocking, but I'd use `const` for all the `var`s